### DISCUSSION TITLE
Release qibb v1.16.0

### DISCUSSION THREAD

**Discussion by vkyosev** [techtriq/qibb#37] [Announcements]
_v1.16.0 was published on 25 August 2022._

<h2>Summary</h2>

* New App Page for flows
  * The App Page for flows just got a fresh overhaul. We have added a responsive layout similar to the look and feel of the catalog.
  * Now you can reach common actions directly from a menu in the top bar. Here you can edit the app details as well as upgrade, rollback or delete the app.
* Various Improvements and bug fixing
  * For better user experience we have made various cosmetic improvements to the catalog pages.
  * To offer better recognition of the flow catalog elements we have added support for two-line titles into that section.
  * Hovering included nodes in the node catalog now unveils their full title.

<hr /><em>This discussion was created from the release <a href='https://github.com/techtriq/qibb/releases/tag/v1.16.0'>v1.16.0</a>.</em>

**Comment by rholzhause** [comment]
That's a very promising update. Pretty sure the users will love it! 🥳
